common/sqliteutil.h

```diff
@@ -0,0 +1,42 @@
+#ifndef ANDEME_COMMON_SQLITE_UTIL_H
+#define ANDEME_COMMON_SQLITE_UTIL_H
+#include <sqlite3.h>
+#include <string>
+#include <vector>
+#include <iostream>
+#include <functional>
+
+#include <schema/message.pb.h>
+
+namespace andeme {
+
+    using Row = std::vector<std::string>;
+    using Callback = std::function<bool(const std::vector<std::string>&)>;
```

Comment:
```suggestion
    using Callback = std::function<bool(const Row&)>;
```
То же самое, но понятнее в чём смысл